iron-focusables-helper.d.ts

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
@valdrinkoshi Is `IronFocusablesHelper` supposed to be something users use, or is it fine that this file is empty? It doesn't have any annotations, so analyzer doesn't pick it up as an interesting feature. Same question for `IronScrollManager`.

Reply:
All managers/helpers are singletons available on `window` and expose public methods, so can be used by users: `IronFocusablesHelper, IronOverlayManager, IronScrollManager`
I'm sure users rely on the public methods of `IronFocusablesHelper` and `IronOverlayManager` as they're been around for a while.

Reply:
I've added some annotations to generate `IronScrollManager`, although it is missing a getter. `IronOverlayManager` is probably too difficult for Analyzer to understand at all right now. Filed https://github.com/PolymerElements/iron-overlay-behavior/issues/262 with details on both.

I'd like to merge this PR despite these missing types, because this element is a dependency of a lot of other ones. I'd like to go back and add some manual typings for the managers a bit later, if that's ok.